### ISSUE TITLE
The command `update prs -fresh` is not adding new PRs

### ISSUE THREAD

**Issue by danillos** [ptsochantaris/trailer-cli#4]
When I run `trailer update prs comments -r {repository} -fresh` it not adding new PRs I had to run `trailer update all -fresh` to the new PRs be added.

**Comment by ptsochantaris** [comment]
Hi @danillos,

Thanks for letting me know. Will check it out asap to see if there is some regression there.

**Comment by ptsochantaris** [comment]
Yep, can confirm this is happening. Updating everything brings in new PRs for e.g. the Apple Swift repo: `trailer update all` while `trailer update prs comments -r swift -fresh` seems to fail to do so. It does detect merged/closed PRs, but won't bring in the new ones (or if it does it nukes them perhaps? Will investigate.)

**Comment by ptsochantaris** [comment]
Interestingly enough, this happens even without the `-fresh` command. It's the `-r` parameter that's causing the new PRs to not show up. Have verified that it's not a display issue - the PRs themselves are got getting stored locally for some reason. Will check this out in more depth soon and keep you posted.

**Comment by ptsochantaris** [comment]
Ah, I forgot the logic of my own app :) From the docs: ```It's useful to remember that you can also use the filtering parameters (that you use when listing items) to also filter items for updates. This will affect only existing items though, and not scan for new ones.```

This does feel a bit wrong though - or rather, it feels as if you ought to be able to check for new items only in a specific repository if you so wish. I know _why_ the filtering options will not fetch new items, it's because the filtering by its nature contradicts the first pass of the queries to the GraphQL endpoint (where the app scans repos and builds an item ID list from them to subsequently fetch). When you add filters this ID list is build by filtering local items.

Obviously you can limit the repos scanned by using the `config` commands, but this is not what you want I think - what you're looking for here is to explicitly not check every enabled repo, but for a single sync, limit the repo list to a certain repo.

Something like `trailer update -prs -comments -from <repo name>` perhaps?

**Comment by ptsochantaris** [comment]
Try this build. It includes the `-from` parameter and I think that works out quite well.

It works like above: `-from` will filter your active repo list during an update. `-r` won't do that, as it's meant to filter local items. That means using it for filtering when updating will work, but will then it will only check for updates to the locally filtered items. I've updated the help text a bit to reflect this.

Let me know how this new parameter works out for you.

Edit: updated binary, added code to skip checking for new repos when `-from` is used, as it would be a bit pointless :)

Edit2: Moved this to release status, as I find myself using it quite a bit, feel free to grab a copy from the releases page.
